docs/changelog/0.3.0.rst

```diff
@@ -0,0 +1,21 @@
+XRTpy v0.3.0 (2023-03-31)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- Modified ``xrt_teem``` to return a `~collections.namedtuple` of SunPy maps. (:pr:`148`)
```

Comment:
```suggestion
- Modified `~xrtpy.response.xrt_teem.xrt_teem` to return a `~collections.namedtuple` of SunPy maps. (:pr:`148`)
```

Reply:
Awesome, I did not know this method create a reference link.